README.md

```diff
@@ -1,57 +1,51 @@
-# Project Name
+# React Calculator
 
-> One paragraph statement about the project.
+> A basic calculator made with Love and React ♥.
 
-![screenshot](./app_screenshot.png)
+![image](https://user-images.githubusercontent.com/5160907/92538253-d88a7e00-f203-11ea-9254-0f28e5bba7d7.png)
 
-Additional description about the project and its features.
+This is the first milestone which is about seting up the project and deploy it to heroku.
 
 ## Built With
 
-- Major languages,
-- frameworks,
-- technologies used
+- React,
+- ES6,
+- NPM
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://calculator-by-enelesmai.herokuapp.com/)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+Get a local copy using 
 
-
-To get a local copy up and running follow these simple example steps.
+        git clone
```

Comment:
You could be more specific in your README file for non-technical individual.

It would be more helpful this way 👇 
![image](https://user-images.githubusercontent.com/31897434/92537386-e5bf6100-f233-11ea-89a2-9729e2bd75fd.png)

Reply:
Updated readme
Thanks for your suggestion bellom! :).